src/test/java/org/dbos/apiary/CrossDBTests.java

```diff
@@ -96,6 +96,45 @@ public void testElasticsearchBasic() throws InvalidProtocolBufferException, Inte
         assertEquals(1, res);
     }
 
+    @Test
+    public void testElasticsearchUpdate() throws InvalidProtocolBufferException, InterruptedException {
+        logger.info("testElasticsearchUpdate");
+
+        ElasticsearchConnection conn;
+        PostgresConnection pconn;
+        try {
+            conn = new ElasticsearchConnection("localhost", 9200, "elastic", "password");
+            pconn = new PostgresConnection("localhost", ApiaryConfig.postgresPort, "postgres", "postgres", "dbos");
+        } catch (Exception e) {
+            logger.info("No Elasticsearch/Postgres instance! {}", e.getMessage());
+            return;
+        }
+
+        apiaryWorker = new ApiaryWorker(new ApiaryNaiveScheduler(), 4);
+        apiaryWorker.registerConnection(ApiaryConfig.elasticsearch, conn);
+        apiaryWorker.registerConnection(ApiaryConfig.postgres, pconn);
+        apiaryWorker.registerFunction("PostgresSearchPerson", ApiaryConfig.postgres, PostgresSearchPerson::new);
+        apiaryWorker.registerFunction("PostgresIndexPerson", ApiaryConfig.postgres, PostgresIndexPerson::new);
+        apiaryWorker.registerFunction("ElasticsearchIndexPerson", ApiaryConfig.elasticsearch, ElasticsearchIndexPerson::new);
+        apiaryWorker.registerFunction("ElasticsearchSearchPerson", ApiaryConfig.elasticsearch, ElasticsearchSearchPerson::new);
+        apiaryWorker.startServing();
+
+        ApiaryWorkerClient client = new ApiaryWorkerClient("localhost");
+
+        int res;
+        res = client.executeFunction("PostgresIndexPerson", "matei", 1).getInt();
+        assertEquals(1, res);
+
+        res = client.executeFunction("PostgresSearchPerson", "matei").getInt();
+        assertEquals(1, res);
+
+        res = client.executeFunction("PostgresIndexPerson", "matei", 2).getInt();
+        assertEquals(2, res);
+
+        res = client.executeFunction("PostgresSearchPerson", "matei").getInt();
+        assertEquals(1, res);
```

Comment:
Is it possible for PostgresSearchPerson to return the actual value of the updated number? So the test can capture if "matei" record is actually updated to 2.